e2e/variantsTest.spec.js

```diff
@@ -64,6 +64,39 @@ import { expect, test } from '@playwright/test'
     ],
     variantId: '23456781',
   },
+  {
+    pageAlt: 'ternary',
+    buttons: [['0', '2', '1', '1', '2', '2', '0', '2', '2', '1', '1', '1', '1', '0', '0', '0', '2', '0', '1', '0', '0']],
+    variantId: 'Trojkovka (hodnoty 0=1, 1=2, 2=0; pořadí 1.=>1., 2.=>2., 3.=>3.; A=0, s Ch)',
+    buttons: [
+      [
+        '0',
```

Comment:
I think it would be more readable to pass array of digits as one string, e.g. `0211...`.